cli/plasmo/src/features/extension-devtools/build-watcher.ts

```diff
@@ -7,6 +7,10 @@ import type { PlasmoManifest } from "~features/manifest-factory/base"
 
 const payload = JSON.stringify({ type: "build_ready" })
 
+const DEBOUNCE_TIME = 740
```

Comment:
What's the justification for 740? I'd love for this to be significantly lower, [ref](https://lawsofux.com/doherty-threshold/)

Reply:
The Parcel's fs debounce is 500ms so thsi is 240ms after (which is a reverse of 42, in turn 74 is a reverse of 47 🤯 )